packages/core/src/__integrationtests__/Ctypes.spec.ts

```diff
@@ -108,42 +101,26 @@ describe('When there is an CtypeCreator and a verifier', () => {
       submitTx(authorizedStoreTx2, paymentAccount)
     ).rejects.toMatchObject({ section: 'ctype', name: 'CTypeAlreadyExists' })
 
-    expect(CType.fromChain(await api.query.ctype.ctypes(ctype.hash))).toBe(
-      ctypeCreator.uri
-    )
+    expect(
+      CType.fromChain(
+        await api.query.ctype.ctypes(CType.getCTypeHashFromId(ctype.$id))
+      )
+    ).toBe(ctypeCreator.uri)
   }, 45_000)
 
   it('should tell when a ctype is not on chain', async () => {
-    const iAmNotThere = CType.fromSchema({
-      $id: 'kilt:ctype:0x2',
-      $schema: 'http://kilt-protocol.org/draft-01/ctype#',
-      title: 'ctype2',
-      properties: {
-        game: { type: 'string' },
-      },
-      type: 'object',
-    } as ICType['schema'])
-
-    const iAmNotThereWithOwner = CType.fromSchema(
-      {
-        $id: 'kilt:ctype:0x3',
-        $schema: 'http://kilt-protocol.org/draft-01/ctype#',
-        title: 'ctype2',
-        properties: {
-          game: { type: 'string' },
-        },
-        type: 'object',
-      },
-      ctypeCreator.uri
-    )
+    const iAmNotThere: ICType = CType.fromProperties('ctype2', {
```

Comment:
Do we need to explicitly mention the type here?

Reply:
no, not necessarily